client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -365,11 +365,21 @@ export default React.createClass( {
 
 	recordSubmit() {
 		const errors = formState.getErrorMessages( this.state.form );
-		analytics.tracks.recordEvent( 'calypso_contact_information_form_submit', {
-			errors,
+		let tracksEventObject = {
```

Comment:
We can merge reduce part and this object into one construct and avoid using `let`:

```js
const tracksEventObject = reduce(
 		formState.getErrorMessages( this.state.form ),
 		( result, value, key ) => {
 			result[ `error_${ key }` ] = value;
 			return result;
 		},
 		{
 		 	errors_count: errors && errors.length || 0,
 		 	submission_count: this.state.submissionCount + 1
 		}
 );
```